ocean-client/addons/command-center/src/utils/store.ts

```diff
@@ -0,0 +1,129 @@
+import SSM from 'aws-sdk/clients/ssm'
+
+// handle AWS Paramter
+export class Store {
+    private ssm: SSM
+    private postfix: string
+    readonly settings: StoredSettings
+
+    constructor() {
+        this.ssm = new SSM()
+        this.postfix = process.env.VAULTMAXI_STORE_POSTFIX ?? process.env.VAULTMAXI_STORE_POSTIX ?? ""
+        this.settings = new StoredSettings()
+    }
+
+    async updateExecutedMessageId(id: number): Promise<unknown> {
+        return this.updateParameter(StoreKey.LastExecutedMessageId, "" + id)
+    }
+
+    async updateSkip(value: boolean = true): Promise<unknown> {
+        return this.updateParameter(StoreKey.Skip, value ? "true" : "false")
+    }
+
+    async updateRange(min: string, max: string): Promise<void> {
+        await this.updateParameter(StoreKey.MinCollateralRatio, min)
+        await this.updateParameter(StoreKey.MaxCollateralRatio, max)
+    }
+
+    async updateReinvest(value: string): Promise<unknown> {
+        return this.updateParameter(StoreKey.Reinvest, value)
+    }
+
+    async updateToken(value: string): Promise<unknown> {
+        return this.updateParameter(StoreKey.LMToken, value)
+    }
+
+    async updateAutoDonation(value: string): Promise<unknown> {
+        return this.updateParameter(StoreKey.AutoDonation, value)
+    }
+
+    async updateStableArbBatchSize(value: string): Promise<unknown> {
+        return this.updateParameter(StoreKey.StableArbBatchSize, value)
```

Comment:
would it make sense to not have a seperate method for each key, but a wrapper `setParameter(key,value)` ? aka: why not just make `updateParameter` public?

Reply:
For me it was just a design decision to avoid needing to know outside the store, which parameter needs to be updated.

Also makes the code more verbose and easier to read what happens.